packages/cli-kit/src/public/node/testing/output.ts

```diff
@@ -1,6 +1,21 @@
-import * as output from '../output.js'
+import * as output from '../../../output.js'
```

Comment:
Could this file be private instead of public? Or is the idea that we want to give adopters of `cli-kit` a tool to help them test in their code as well?

Reply:
It has to be public because it's used by other packages like `cli`.